test/fs/test_idbfs_autopersist.c

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright 2025 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and the
+ * University of Illinois/NCSA Open Source License.  Both these licenses can be
+ * found in the LICENSE file.
+ */
+
+#include <assert.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <emscripten.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <sys/stat.h>
+#include <errno.h>
+
+EM_JS_DEPS(deps, "$callUserCallback");
+
+void finish() {
+  emscripten_force_exit(0);
+}
+
+static void prepare() {
+  printf("cleaning up test files\n");
+  unlink("/working1/file");
+  unlink("/working1/file_renamed");
+  unlink("/working1/symlink");
+  rmdir("/working1/dir");
+}
+
+void test() {
+#ifdef TEST_PREPARE
+  prepare();
+  goto end;
+#endif
```

Comment:
Instead of `goto end` here can you just use `#else` and have `#endif` instead of the end label?